### ISSUE TITLE
Heroic Strikes Milestone content not visible

### ISSUE THREAD

**Issue by hskrasek** [Bungie-net/api#326]
Just a quick note, noticed that the heroic strikes milestone Id 3405519164 is showing up in the milestones list. But if I tried to get the milestone content, it just kicks back an empty response.

The manifest shows the data, so it doesn't seem to be an issue with it being classified.

**Comment by vthornheart-bng** [comment]
Ah!  Thank you, I will investigate!

**Comment by vthornheart-bng** [comment]
Oh, you're talking the separate endpoint, the one that returns localized content?  Just making sure.

**Comment by hskrasek** [comment]
@vthornheart-bng Correct. If I call `https://www.bungie.net/Platform/Destiny2/Milestones/3405519164/Content/` I get the following response:

```json
{
  "ErrorCode": 1,
  "ThrottleSeconds": 0,
  "ErrorStatus": "Success",
  "Message": "Ok",
  "MessageData": {}
}
```

**Comment by vthornheart-bng** [comment]
Ahhh, cool, good deal!  Yes, we will have to get that content loaded: usually we don't provide the additional localized content except for items that appear on the "Explore Destiny" section of the website.  But that is an item that we actually intended to put there and haven't yet!  Soon it will have the kind of extra localized content you are looking for (ideally before mid-month, unless we get delayed)

**Comment by hskrasek** [comment]
@vthornheart-bng Ahh gotcha, makes total sense. Well it's a good thing I'm working on migrating my little app rely on both the API and the manifest file in the mean time. Glad I could help!

**Comment by vthornheart-bng** [comment]
Oh yes, definitely a good idea!  If I had to give a rough estimate, I'd say maybe 20% of the milestones that exist today actually have localized content.

**Comment by hskrasek** [comment]
@vthornheart-bng Gotcha. So far I've been using the content api and the list api to build out a weekly milestone list like in the game, but as a Slack message. Has worked so far, but looking at the app I can see the descriptions of milestones differ. 

But after running the app through Charles proxy I figured out the app is just using the "list milestones" endpoint to get what is active, and then pulling everything else from the manifest. I'd make that update today but.. I mean.. I gotta go play Curse of Osiris first.

**Comment by vthornheart-bng** [comment]
Sweet, and understandable! :)
